sys/include/net/ng_pkt.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    ng_pkt Packet
+ * @brief       Network packet abstraction types
+ * @ingroup     net
+ * @{
+ *
+ * @file
+ * @brief   General definitions for network packets
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef __NG_PKT_H_
+#define __NG_PKT_H_
```

Comment:
pls rename to `NG_PKT_H_` as prepended underscores are not standard conform (as I newly learned :-) )